src/main/java/org/kohsuke/github/GHTeamBuilder.java

```diff
@@ -0,0 +1,94 @@
+package org.kohsuke.github;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Creates a team.
+ *
+ * https://developer.github.com/v3/teams/#create-team
+ */
+public class GHTeamBuilder {
+
+    private final GitHub root;
+    protected final Requester builder;
+    private final String orgName;
+
+    public GHTeamBuilder(GitHub root, String orgName, String name) {
+        this.root = root;
+        this.orgName = orgName;
+        this.builder = root.createRequest();
+        this.builder.with("name", name);
+    }
+
+    /**
+     * Description for this team.
+     *
+     * @param description
+     *            description of team
+     * @return a builder to continue with building
+     */
+    public GHTeamBuilder description(String description) {
+        this.builder.with("description", description);
+        return this;
+    }
+
+    /**
+     * Maintainers for this team.
+     *
+     * @param maintainers
+     *            maintainers of team
+     * @return a builder to continue with building
+     */
+    public GHTeamBuilder maintainers(List<String> maintainers) {
```

Comment:
Question: 
```suggestion
    public GHTeamBuilder maintainers(String... maintainers) {
```

Reply:
I mean these suggestions as added convenience methods. ... But maybe they can wait to a future release.

Reply:
makes sense, no annoying singletonList to use it